File: PHP Excel Examples/WorksheetsApiDemo/renameWorksheet.php
<?php

use Spire\Cloud\Excel\Sdk\Configuration;

$appId = "your id";
$appKey = "your key";
$baseUrl = "https://api.e-iceblue.cn";
$configuration = new Configuration($appId, $appKey, $baseUrl);
$apiInstance = new WorksheetsApi($configuration);

$name = "RenameWorksheet_Text.xlsx";
$folder = "input";
$storage = null;
$sheetName = "Sheet2";
$newname = "newSheetName";
$result = $apiInstance->renameWorksheet($name, $sheetName,$newname,$folder,$storage);

File: Python Excel Examples/WorksheetsApiDemo/renameWorksheet.py
import spirecloudexcel
from spirecloudexcel.configuration import Configuration as ExcelConfiguration

appId = "your id"
appKey = "your key"
baseUrl="https://api.e-iceblue.cn"
configuration = ExcelConfiguration(appId, appKey,baseUrl)
api = spirecloudexcel.api.worksheets_api.WorksheetsApi(configuration)

name = "RenameWorksheet.xlsx"
folder = "ExcelDocument";
storage = ""
sheetName = "Sheet1"
newName = "newSheetName"
api.rename_worksheet(name, sheetName, newName, folder=folder, storage=storage)